docs/actions/refresh-grid.md

```diff
@@ -0,0 +1,43 @@
+---
+id: refresh-grid
+title: Refresh Action Grid
+sidebar_label: Refresh Action Grid
+---
+
+> Audience: [`Citizen Developers`](/docs/audience#citizen-developers)<br/>
+> Skill Prerequisites: `Using Action Grid`
+
+This action is usually used to refresh the grid after executing another action.
+
+## `Typical Use Cases`
+
+- Refresh Action Grid with updated without reloading the entire page
+
+## `Actions Covered by this Document`
```

Comment:
I only started adding this section with the last couple PRs because we had docs covering more than on action. When a doc only covers one action, do you think we should include for consistency? If so, we will need to go back and update the older action docs.

Reply:
I wasn't sure about this either since it is just the one action.  I'm leaning toward leaving it off if there is only one action being covered cause it doesn't make as much sense.  It only shows me the things you changed in this particular view so I'll figure out how to remove it.  Don't let it go through until I figure it out please :)